rviz_common/include/rviz_common/selection/selection_handler.hpp

```diff
@@ -41,9 +41,9 @@
 
 #include <OgreMovableObject.h>
 
-#include "../interactive_object.hpp"
+#include "src/rviz_common/interactive_object.hpp"
 #include "rviz_common/properties/property.hpp"
-#include "../viewport_mouse_event.hpp"
+#include "src/rviz_common/viewport_mouse_event.hpp"
```

Comment:
This doesn't look right. A public header cannot reference a file in the src folder.